examples/client/Locomotion/src/services/network.js

```diff
@@ -68,11 +68,14 @@ class Network {
           AppSettings.getOperationId(),
         ]);
         this.axios.defaults.baseURL = baseURL;
-        const accessToken = await Auth.getAT(this.axios);
+        const [accessToken, captchaToken] = await Promise.all([
+          Auth.getAT(this.axios),
+          Config.CAPTCHA_KEY && Auth.getCaptchaToken(),
+        ]);
         this.axios.defaults.headers.common.Authorization = accessToken ? `Bearer ${accessToken}` : accessToken;
-
         this.axios.defaults.headers.common['x-loco-ds-id'] = operationId;
         this.axios.defaults.headers.common['x-loco-op-id'] = operationId;
+        this.axios.defaults.headers.common['x-captcha-token'] = captchaToken;
```

Comment:
do we want to add it for every request?
as I understand its generate it only on the login right?

Reply:
adjusted it so only if no access token we add it , that way it will be used only on the relevant endpoints